config/_default/menus/menus.en.yaml

```
@@ -1191,11 +1191,11 @@ main:
     parent: slos
     identifier: burn_rate
     weight: 404
-  - name: Incident Management
-    url: monitors/incident_management
+  - name: Case Management
+    url: monitors/case_management
     parent: alerting
-    identifier: incidents
-    weight: 5
+    identifier: cases
```

If this doesn't have to match the UI Url, might be worth being explicit for future clarity?

```suggestion
    identifier: case_management
```